tests/phpunit/tests/media.php

```diff
@@ -6445,6 +6445,55 @@ public function test_wp_img_tag_add_auto_sizes( string $input, string $expected
 		);
 	}
 
+	/**
+	 * @ticket 62305
+	 *
+	 * @dataProvider data_provider_to_test_image_metadata_get_converted_image_in_file_key
+	 *
+	 * @param bool $apply_big_image_size_threshold True if filter needs to apply, otherwise false.
+	 */
+	public function test_image_metadata_get_converted_image_in_file_key( bool $apply_big_image_size_threshold ) {
```

Comment:
Maybe more clear as:
```suggestion
	public function test_image_converted_to_other_format_has_correct_filename( bool $apply_big_image_size_threshold ) {
```